src/components/projects/projects-table/projects-table-filter/projects-table-filter.test.tsx

```diff
@@ -64,16 +64,16 @@ describe('projects table filter component', () => {
   it('checking if data in the status dropdown menu is correct', async () => {
     renderComponent();
     expect(screen.queryByText('None')).not.toBeInTheDocument();
-    expect(screen.queryByText('Active')).not.toBeInTheDocument();
-    expect(screen.queryByText('Inactive')).not.toBeInTheDocument();
-    expect(screen.queryByText('Complete')).not.toBeInTheDocument();
+    expect(screen.queryByText('ACTIVE')).not.toBeInTheDocument();
```

Comment:
to make these more flexible, you should be using `WbsElementStatus.Active` and the associated others for all these edited lines.